python/ray/autoscaler/gcp/example-gpu-docker.yaml

```diff
@@ -74,7 +74,7 @@ available_node_types:
                 - key: install-nvidia-driver
                   value: "True"
             scheduling:
-              - onHostMaintenance: TERMINATE
+              onHostMaintenance: TERMINATE
```

Comment:
Other places in the file is still `- onHostMaintenance:`, could you do a global search and replace all of them?

Reply:
@jjyao  thanks, done.